### ISSUE TITLE
SyntaxError: Unexpected token > at Module._compile (module.js:439:25)

### ISSUE THREAD

**Issue by thijssmudde** [zodern/meteor-up#729]
Deployment error with the latest version. See here the logs:

```
npm WARN deprecated This version of npm lacks support for important features,
npm WARN deprecated such as scoped packages, offered by the primary npm
npm WARN deprecated registry. Consider upgrading to at least npm@2, if not the
npm WARN deprecated latest stable version. To upgrade to npm@2, run:
npm WARN deprecated 
npm WARN deprecated   npm -g install npm@latest-2
npm WARN deprecated 
npm WARN deprecated To upgrade to the latest stable version, run:
npm WARN deprecated 
npm WARN deprecated   npm -g install npm@latest
npm WARN deprecated 
npm WARN deprecated (Depending on how Node.js was installed on your system, you
npm WARN deprecated may need to prefix the preceding commands with `sudo`, or if
npm WARN deprecated on Windows, run them from an Administrator prompt.)
npm WARN deprecated 
npm WARN deprecated If you're running the version of npm bundled with
npm WARN deprecated Node.js 0.10 LTS, be aware that the next version of 0.10 LTS
npm WARN deprecated will be bundled with a version of npm@2, which has some small
npm WARN deprecated backwards-incompatible changes made to `npm run-script` and
npm WARN deprecated semver behavior.
npm WARN package.json meteor-dev-bundle@0.0.0 No description
npm WARN package.json meteor-dev-bundle@0.0.0 No repository field.
npm WARN package.json meteor-dev-bundle@0.0.0 No README data
npm WARN package.json string_decoder@1.0.3 string_decoder is also the name of a node core module.
npm WARN cannot run in wd meteor-dev-bundle@0.0.0 node npm-rebuild.js (wd=/bundle/bundle/programs/server)
=> Starting meteor app on port:80

/bundle/bundle/programs/server/mini-files.js:82
const unicodeNormalizePath = (path) => {
                                     ^
SyntaxError: Unexpected token >
    at Module._compile (module.js:439:25)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/bundle/bundle/programs/server/boot.js:9:13)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
```

**Comment by thijssmudde** [comment]
mup.js

```
module.exports = {
  servers: {
    one: {
      host: '146.185.158.94', //myroutes.io
      username: '',
      password: ''
    }
  },
  meteor: {
    name: 'myroutes',
    path: '../myroutes',
    servers: {
      one: {},
    },
    buildOptions: {
      serverOnly: true,
    },
    env: {
      ROOT_URL: 'https://myroutes.io',
      MONGO_URL: 'mongodb://localhost/meteor',
    },
    ssl: {
      crt: './myroutes_io.crt', // this is a bundle of certificates
      key: './private.key', // this is the private key of the certificate
      port: 443 // 443 is the default value and it's the standard HTTPS port
    },
    dockerImage: 'abernix/meteord:base',
    deployCheckWaitTime: 60,
    enableUploadProgressBar: true
  },
  mongo: {
    oplog: true,
    port: 27017,
    version: '3.4.1',
    servers: {
      one: {},
    },
  },
};
```

**Comment by thijssmudde** [comment]
Mup setup also returns an error:

	docker: Error response from daemon: Conflict. The container name "/myroutes-dummy-http-server-for-ssl-check" is already in use by container 3cd78b081f65b945a7d2c1b69046204c721f92c9cbe3a72fa6e314dda1c7653b. You have to remove (or rename) that container to be able to reuse that name..
	See 'docker run --help'.
	docker: Error response from daemon: Cannot link to a non running container: /myroutes-dummy-http-server-for-ssl-check AS /cranky_poitras/backend.
	time="2017-09-09T19:55:02Z" level=error msg="error getting events from daemon: context canceled"

**Comment by houssainy** [comment]
I have the same issue after i upgraded to mup 1.3.2, this error is keep appearing to me in Verifying Deployment step and i tried all possible solutions but it didn't work until i downgraded to 1.3.1

So i think this error related to mup 1.3.2

**Comment by zodern** [comment]
The next time someone has this error, what does `mup docker ps` show?

**Comment by dphov** [comment]
@zodern 
mup.js
```js
module.exports = {
  servers: {
    one: {
      // Production
      host: 'iphere',
      username: 'root',
      password: 'password',
      // opts: {
      //   port: 22
      // }
    }
  },
  meteor: {
    name: 'app',
    path: '..',
    dockerImage: 'abernix/meteord:base',
    servers: {
      one: {}
    },
    buildOptions: {
      // serverOnly: true,
      debug: true,
      executable: 'meteor'
    },
    env: {
      PORT: 3000,
      ROOT_URL: 'https://app.com',
      MONGO_URL: 'mongodb://localhost/meteor'
    },
    deployCheckWaitTime: 45,
    enableUploadProgressBar: true,
    ssl: {
      autogenerate: {
        email: 'mail@example.com',
        domains: 'example.com,www.example.com'
      }
    }
  },
  mongo: {
    oplog: true,
    port: 27017,
    servers: {
      one: {}
    }
  }
};
```

`mup docker ps`

CONTAINER ID | IMAGE | COMMAND | CREATED | STATUS | PORTS 
------------ | ------------ | ------------ | ------------ |------------ |------------
|71082dc86313 | jrcs/letsencrypt-nginx-proxy-companion:latest |"/bin/bash /app/en..." |  57 seconds ago|Up 56 seconds |   | 
6675cfb3dc83 | jwilder/nginx-proxy:latest | "/app/docker-entry..." | About a minute ago  | Up About a minute | 0.0.0.0:80->80/tcp, 0.0.0.0:443->443/tcp  
9f6423882449     |   kadirahq/meteord                   |             "/bin/sh -c 'bash ..."  | About a minute ago |  Restarting (8) 2 seconds ago |          |                           
56ca7ffeafda   |     mongo:3.4.1              |                       "/entrypoint.sh mo..." |  21 minutes ago       | Up 21 minutes        |          127.0.0.1:27017->27017/tcp

**Comment by zodern** [comment]
It's using the wrong docker image. Both configs have the docker image set in `meteor.dockerImage`. Try setting it in `meteor.docker.image` instead. I will try to fix it for the next release.

**Comment by zodern** [comment]
`meteor.dockerImage` will be fixed in 1.3.3.

> docker: Error response from daemon: Conflict. The container name "/myroutes-dummy-http-server-for-ssl-check" is already in use by container

This will also be fixed in 1.3.3.
